docs/REST API/rest-api.md

```diff
@@ -8926,6 +8949,7 @@ Returns information about the specified software. By default, `versions` are sor
   "software_title": {
     "id": 12,
     "name": "Firefox.app",
+    "bundle_identifier": "org.mozilla.firefox",
     "software_package": {
```

Comment:
#### Dev note:

For VPP apps `software_package` will be `null` and new object will be there instead (`app_store_app`). See in [Figma](https://www.figma.com/design/QsO3Q03QGVbiSknGpLqgAB/%2318867-Install-Apple-App-Store-apps-on-macOS?node-id=494-9763&t=QY0QlnuIig1o8gZN-11).